fossunited/foss_profiles/doctype/foss_user_profile/foss_user_profile.py

```diff
@@ -313,8 +314,34 @@ def get_context(self, context):
             context.volunteered,
         ) = self.get_user_activity()
 
+        context.pagetitle, context.description, context.image = self.get_meta()
+
         context.no_cache = 1
 
+    def get_meta(self):
+        if self.is_private:
+            return self.username, "Private Profile", ""
+
+        # eg. Arya | arya_k
+        pagetitle = self.full_name + " | " + self.username
+
+        desc_short = ""
+        if self.about:
+            desc_short = textwrap.shorten(re.sub(r"<.*?>", "", self.about), width=150)
+
+        description = "{self.full_name} is a FOSS United Community Member. {desc_short}".format(  # noqa: E501
```

Comment:
The URL will already say fossunited.org, so any specific reason why we want to add the "... is a FOSS United Community member" - it's extra text, and I'm not sure what additional information it conveys?